Chaincase.UI/ViewModels/SendViewModel.cs

```diff
@@ -404,6 +408,40 @@ public async Task<bool> SendTransaction(string password)
             return false;
         }
 
+        private static IPayjoinClient? GetPayjoinClient(string payjoinEndPoint, bool isTorEnabled, Network network, EndPoint torSocks5EndPoint)
+        {
+            if (string.IsNullOrWhiteSpace(payjoinEndPoint) ||
+                !Uri.IsWellFormedUriString(payjoinEndPoint, UriKind.Absolute))
+            {
+                return null;
```

Comment:
Would it make more sense to throw? Instead of returning null, the caller would have an exception to work with.

Reply:
normally I would say yes, however the SendViewModel probably shouldn't even know about PayJoinClient's workings to handle exceptions in the first place.

If you agree, and believe this implementation mirrors the existing Wasabi behavior I propose we ship this as-is.

The in-progress refactor uses BtcPayServer.BIP78.Sender library, constructs PayjoinClient via DI and has proper exception handling. Your concern is being addressed elsewhere.

Reply:
Sounds good!